.circleci/config.yml

```diff
@@ -571,7 +571,7 @@ jobs:
             export PULL_TAG="${COMMIT}"
             export TAG="<< parameters.tag >>"
             export REPO="networkservicemesh"
-            export CONTAINERS=(nsmd nsmd-k8s nsmdp crossconnect-monitor nsm-init vppagent-dataplane vppagent-dataplane-dev admission-webhook)
+            export CONTAINERS=(nsmd nsmd-k8s nsmdp crossconnect-monitor nsm-init vppagent-dataplane vppagent-dataplane-dev admission-webhook vppagent-icmp-responder-nse vppagent-nsc nsc icmp-responder-nse vppagent-firewall-nse vppagent-passthrough-nse vpn-gateway-nse vpn-gateway-nsc)
```

Comment:
How about layering these for better visibility:

```shell
            export CONTAINERS=(nsmd nsmd-k8s nsmdp)
            export CONTAINERS=(${CONTAINERS[@]} crossconnect-monitor nsm-init admission-webhook)
            export CONTAINERS=(${CONTAINERS[@]} vppagent-dataplane vppagent-dataplane-dev)
            export CONTAINERS=(${CONTAINERS[@]} vppagent-icmp-responder-nse vppagent-nsc)
            export CONTAINERS=(${CONTAINERS[@]} nsc icmp-responder-nse)
            export CONTAINERS=(${CONTAINERS[@]} vppagent-firewall-nse vpn-gateway-nse vpn-gateway-nsc)
```

Note that there is no `vppagent-passthrough-nse`. Also at some point we're going to supplement some fo these containers from the examples repo.

Reply:
from a user perspective its definitely more readable